trunk/SUAVE/Input_Output/Results/plot_weight_comparison.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import warnings
+from SUAVE.Core import Units
+from matplotlib.pyplot import cm
+
+
+def plot_weight_comparison(lst_files, caseNames=[], units='kg'):
+    if len(caseNames) < len(lst_files):
```

Comment:
The functions need the standard SUAVE headers

Reply:
FINC